src/components/GroupSuggestions/SuggestionBox/index.js

```diff
@@ -1,24 +1,22 @@
-import React, { memo, useState, useEffect } from 'react'
+import React, { memo } from 'react'
 import './styles.scss'
 import propTypes from 'prop-types'
-import { Link } from 'react-router-dom'
 
 const SuggestionBox = props => {
-  const { channel, handleJoinChannel, getMembersByChannelId } = props
-  const [members, setMembers] = useState([])
-
-  useEffect(() => {
-    getMembersByChannelId(channel.id).then(res => setMembers(res))
-  }, [])
+  const { channel, members, hidden, top } = props
 
   return (
-    <div className="group-box suggestions-box">
+    <div
+      className={`group-box group-suggestion-box ${hidden ? 'hidden-box' : ''}`}
+      aria-hidden={hidden}
+      style={{ top: `-${top + 2}px ` }}
+    >
       <div className="group-box-content">
         <div className="group-header">
           <h2>{channel.display_name}</h2>
         </div>
         <p>{`Yhteistä: ${channel.display_name}`}</p>
-        {members && (
+        {channel && members && (
           <div className="suggestion-members-wrapper">
             <p>{`${members.length} jäsentä`}</p>
             <div className="suggestion-members-info">
```

Comment:
A longer list of channel members' names overflows the container. Maybe change the suggestion-members-info class flex-flow property to wrap?

Reply:
Good one! Fix'd :)